odoo/models.py

```diff
@@ -2904,6 +2872,8 @@ def _add_sql_constraints(self):
     def _execute_sql(self):
         """ Execute the SQL code from the _sql attribute (if any)."""
         if hasattr(self, "_sql"):
```

Comment:
Would probably be a good idea to also add a metaclass checker for people overriding `_execute_sql`, in case anyone found it a useful hook. Because I expect we'll be removing the method as well as the attribute.

Reply:
Yes, good point, I will add it.